<launch>
 

	<node pkg="cob_3d_mapping_common" type="shape_visualization" name="shape_visualization" output="screen">
  	
    <remap from="shape_array" to="/plane_extraction/shape_array"/>
     <remap from="shape_array" to="/shapes_array"/>
     <remap from="shape" to="shape_visualization/shape"/>
  	<!--remap from="point_cloud" to="/cam3d/rgb/points"/-->
  	<remap from="point_cloud" to="/cam3d/depth/points"/>
  	<remap from="point_cloud_out" to="shape_visualization/point_cloud"/>
  	<remap from="marker" to="shape_visualization/marker"/>
  	<remap from="interactive_marker" to="shape_visualization/interactive_marker"/>
	</node>
		
		
</launch>	